devimint/src/vars.rs

```diff
@@ -104,6 +105,7 @@ declare_vars! {
 
         FM_CLN_EXTENSION_LISTEN_ADDRESS: String = "0.0.0.0:8177";
         FM_GATEWAY_LIGHTNING_ADDR: String = "http://localhost:8177";
+        FM_FAUCET_BIND_ADDR: String = "0.0.0.0:15243";
```

Comment:
port number generated by mashing buttons.

Reply:
We might go with something more memorable if we expect users to want to type it in their browser. `19999` or something?